src/vcf/record.cpp

```diff
@@ -472,6 +470,19 @@ namespace ebi
         }
     }
 
+    void Record::check_no_duplicates(std::vector<std::string> const & values) const
+    {
+        if (values.size() > 1) {
+            std::map<std::string, int> counter;
+            for (auto & value : values) {
+                counter[value]++;
+                if (counter[value] >= 2) {
+                    raise(std::make_shared<Error>(line, " must not have duplicate "));
```

Comment:
The way this message needs to be used from other methods is a bit ugly. What about just throwing an `std::invalid_argument`, like in `check_value_type`, then build the whole message in the other methods?